armi/utils/plotting.py

```diff
@@ -1283,10 +1290,10 @@ def _makeComponentPatch(component, position, cold):
     elif isinstance(component, Hexagon):
         if component.getDimension("ip", cold=cold) != 0:
```

Comment:
The changes that you made are only inside of this `if` clause. This means the rotation will only be applied to the hexes that have an `ip`. 

The other major hexagon that doesn't have an `ip` is the `DerivedShape` that often represents the coolant. This one doesn't get rotated by the logic that you've implemented. Here is a diagram that I've produced using your changes:
![image](https://github.com/terrapower/armi/assets/21959600/deb26d2f-cc1a-405c-b058-b8329b1ed5e2)

Reply:
The pictures appear to be fixed for me as well.